src/client.ts

```diff
@@ -76,7 +76,7 @@ export class Client {
 
   subscribeTicker(symbol: string): void {
     this.requireSocketToBeOpen();
-    const formatSymbol = symbol.replace('/', '-');
+    const formatSymbol = symbol.split('/').join('-');
```

Comment:
not a big fan of that change, for example if someone call:

```js
kucoinWs.subscribeTickers(['BTC/USDT', 'ETH/USDT']);
```
it'll store in `subscriptions` array: `ticker-BTC/USDT,ETH/USDT`

So if someone want just to unsubscribe the ticker for: `ETH/USDT` and **not** `BTC/USDT` it won't work
because this subscription won't be found in the `subscriptions` array.
Other use case where it's not the best, if someone later want to just subscribe (again) on `ETH/USDT` it'll create a new connection

Reply:
this comment isn't resolve @TylerSanford